waypoints:
  wp0:
    position:
      x: -2.0
      y: -0.5
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 1.0
      w: 0.0
  wp_table_1:
    position:
      x: 0.84
      y: 0.76
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.73
      w: 0.68
  wp_table_2:
    position:
      x: -0.56
      y: -1.63
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.69
      w: 0.72
  wp_table_3:
    position:
      x: 1.0
      y: -1.86
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.0 
      w: 1.0
  wp_table_4:
    position:
      x: -0.16
      y: 0.76
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 1.0 
      w: 0.0

#fill in the correct waypoints for the cabinets and uncomment      
  wp_cabinet_1:
    position:
      x: 0.48
      y: -1.24
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.707
      w: 0.707
  wp_cabinet_2:
    position:
      x: -0.4356
      y: -1.24
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.707
      w: 0.707
